### PR TITLE
Add robot file to create testcase for exporting data to backend

### LINKED ISSUE #61: Create test cases for exporting data to backend
- Export data to HTTP Server
- Export data to MQTT Server

### PR DESCRIPTION
**Pull request by cherrycl**
- Export data to HTTP Server
- Export data to MQTT Server

Signed-off-by: Cherry Wang <cherry@iotechsys.com>

### CODE REVIEW

File: TAF/testScenarios/integrationTest/UC_end_to_cloud/export_data_to_backend.robot [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    When Create device with autoevent
+    Then Events/readings have exported to HTTP Server
+    And The exported data existed on core-data and mark as Pushed
+
+Export002 - Export events/readings to MQTT Server
+    Given Deploy device service
+    And Deploy configurale application service
+    And Register MQTT Server
+    When Create device with autoevent
+    Then Events/readings have exported to HTTP Server
```
Comment (cloudxxx8) [inline TAF/testScenarios/integrationTest/UC_end_to_cloud/export_data_to_backend.robot]: this should be MQTT

File: TAF/testScenarios/integrationTest/UC_end_to_cloud/export_data_to_backend.robot [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    And The exported data existed on core-data and mark as Pushed
+
+ExportErr001 - Export events/readings to unreachable backend
+    Given Deploy device service
+    And Deploy configurale application service
+    And Register a unreachable backend
+    When Create device with autoevent
+    Then The exported data existed on core-data and doesn't mark as Pushed
+
```
Comment (cloudxxx8) [inline TAF/testScenarios/integrationTest/UC_end_to_cloud/export_data_to_backend.robot]: remove the additional space

File: TAF/testScenarios/integrationTest/UC_end_to_cloud/export_data_to_backend.robot [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    And Deploy configurale application service
+    And Register MQTT Server
+    When Create device with autoevent
+    Then Events/readings have exported to HTTP Server
+    And The exported data existed on core-data and mark as Pushed
+
+ExportErr001 - Export events/readings to unreachable backend
+    Given Deploy device service
+    And Deploy configurale application service
+    And Register a unreachable backend
```
Comment (cloudxxx8) [inline TAF/testScenarios/integrationTest/UC_end_to_cloud/export_data_to_backend.robot]: the backend should be both HTTP and MQTT

**Comment by jpwhitemn** [comment]
@cloudxxx8 and @cherrycl 
In general, I think you have started at the right place and have outlined the right initial export tests.I think there are some things we'd like to see as part of these tests, and a lot of edge cases could be explored, but they don't have to be in the initial implementation; they could be added as issues for later implementations.
1) are we getting the same number of event/readings through EdgeX from the device as expected
2) did the data originating at the DS match that which exports out
3) did the record get persisted
4) did the log entries match what was expected

File: TAF/testScenarios/integrationTest/UC_end_to_cloud/export_data_to_backend.robot [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+${SUITE}         Export data to backend
+
+
+*** Keywords ***
+
+
+*** Test Cases ***
+Export001 - Export events/readings to HTTP Server
+    Given Deploy device service
+    And Deploy configurale application service
```
Comment (cloudxxx8) [inline TAF/testScenarios/integrationTest/UC_end_to_cloud/export_data_to_backend.robot:14]: typo, it's **Configurable**
